docs/rst/tutorials/core/deployment/ds_locators_runtime/ds_locators_runtime.rst

```diff
@@ -0,0 +1,369 @@
+address, port and prefix) in all *clients*.
+The port and prefix are already known, and each specific *server* IP address can be obtained introducing the container
+names in the following command:
+
+.. tabs::
+
+    .. tab:: Discovery Environment 0
+
+        .. code-block:: bash
+
+            docker inspect --format '{{ .NetworkSettings.IPAddress }}' server_0
+
+    .. tab:: Discovery Environment 1
+
+        .. code-block:: bash
+
+            docker inspect --format '{{ .NetworkSettings.IPAddress }}' server_1
+
+.. note::
+
+    For this example, these are the used IP addresses for each entity.
+
+    +----------------+------------+
+    | CONTAINER NAME | IP ADDRESS |
+    +================+============+
+    | client_sub     | 172.17.0.2 |
+    +----------------+------------+
+    | client_pub_0   | 172.17.0.3 |
+    +----------------+------------+
+    | client_pub_1   | 172.17.0.4 |
+    +----------------+------------+
+    | server_0       | 172.17.0.5 |
+    +----------------+------------+
+    | server_1       | 172.17.0.6 |
+    +----------------+------------+
+
+Create a XML configuration file in each *client* container, and complete it with the following XML profile, including
+the *server* data obtained (IP address, port and GUID prefix).
+
+.. literalinclude:: /resources/tutorials/core/deployment/ds_locators_runtime/client_configuration.xml
+    :language: xml
+
+The entities need to be configured to use the created profile.
+To do so, make sure the document name is ``DEFAULT_FASTRTPS_PROFILES.xml``, and set the following environment variable in all *clients*:
+
+.. code-block:: bash
+
+    export RMW_FASTRTPS_USE_QOS_FROM_XML=1
+
+``FASTDDS_ENVIRONMENT_FILE``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Finally, each *client* needs to start a discovery client.
+To do so, and this is the key of the tutorial, the environment variable ``FASTDDS_ENVIRONMENT_FILE`` must be set.
+
+Setting this environment variable to an existing ``json`` file allows to load the environment variables from a file
+instead of from the environment.
+This allows to change the value of some environment variables at run time with just modifying and saving the changes
+to the file.
+
+Create a json file for each *client* and introduce the *server* IP address and port.
+Make sure the talker in the discovery environment 1 only contains its discovery server data:
+
+.. tabs::
+
+    .. tab:: Discovery Environment 0
+
+        .. literalinclude:: /resources/tutorials/core/deployment/ds_locators_runtime/environment_file_complete.json
+            :language: xml
+
+    .. tab:: Discovery Environment 1
+
+        .. literalinclude:: /resources/tutorials/core/deployment/ds_locators_runtime/environment_file.json
+            :language: xml
+
+The environment value can be either an absolute or relative path.
+Set up the environment file:
+
+.. code-block:: bash
+
+    export FASTDDS_ENVIRONMENT_FILE="environment_file.json"
```

Comment:
I'd actually add this line to all the following code blocks (when needed), so users can copy-paste them. Also, it'd be more explicit what each terminal is meant to be doing